Lesson04.Methods/Lesson04.Methods/Program.cs

```diff
@@ -116,5 +119,64 @@ static int SumNumbers(int a, int b)
 
             return sum;
         }
+
+        //------------------------------HOMEWORK---------------------------------//
+
+        static void Master()
+        {
+            Console.Write("Input first number: ");
+            int a = int.Parse(Console.ReadLine());
+            Console.Write("Input second number: ");
+            int b = int.Parse(Console.ReadLine());
+
+            Console.Write("Input string: ");
+            string x = Console.ReadLine();
+
+            MaxMinValue(54, 76);
+            MaxMinValue(3, 43, 23);
+            MaxMinValue(4.3, 9.23, 1.11, 13.32);
+
+            Repeat(x);
+
+            if (TrySumIfOdd(a, b, out int sum))
+            {
+                Console.WriteLine("\nParity \nSum is: " + sum);
+            }
+
+            else
+            {
+                Console.WriteLine("\nNot Parity \nSum is: " + sum);
+            }
+
+        }
+        static void MaxMinValue(int x, int y)
+        {
+            Console.WriteLine("Max value is: " + Math.Max(x, y));
+            Console.WriteLine("Min value is: " + Math.Min(x, y));
+        }
+
+        static void MaxMinValue(int x, int y, int z)
+        {
+            Console.WriteLine("\nMax value is: " + Math.Max(x, Math.Max(y, z)));
+            Console.WriteLine("Min value is: " + Math.Min(x, Math.Min(y, z)));
+        }
+
+        static void MaxMinValue(double x, double y, double z, double w)
+        {
+            Console.WriteLine("\nMax value is: " + Math.Max(x, Math.Max(y, Math.Max(z, w))));
+            Console.WriteLine("Min value is: " + Math.Min(x, Math.Min(y, Math.Min(z, w))));
+        }
+
+        static bool TrySumIfOdd(int a, int b, out int sum)
+        {
+            sum = a + b;
+            return sum % 2 == 0;
```

Comment:
You must calculate sum between numbers, but not just the sum of two of them

Reply:
Fix this part